### PR TITLE
[ISSUE-2988][Bug] Update helm template, fix streampark on k8s failed 

### LINKED ISSUE #2988: [Bug] update helm template, fix streampark on k8s failed
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-streampark/issues?q=is%3Aissue+label%3A%22bug%22) and found no similar issues.

### Java Version

java version：1.8

### Scala Version

2.12.x

### StreamPark Version

streampark version：2.1.1

### Flink Version

flink version： 1.14.5

### deploy mode

kubernetes-application

### What happened

When using streampark on k8s, there are some bugs in the helm templates. I have added some remarks that may need attention.

### Error Exception

_No response_

### Screenshots

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!(您是否要贡献这个PR?)

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by chenshaojin**
## What changes were proposed in this pull request

Issue Number: close #2988 

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

1、The ingress does not have a namespace.
2、The ingress may need to specify the ingress controller.
3、A 404 error occurs when accessing swagger.
4、The startup command cannot use startup.sh because it is nohup. You need to use ./streampark.sh start_docker
5、The resources attribute must be added.
6、add env. Solve the docker log garbled code problem.
7、dind uses tcp:xxxx:2375, which is a bit more convenient.
8、use ingress. The front-end code needs to be recompiled. Modify the .env.production file to change VITE_PUBLIC_PATH and VITE_GLOB_API_URL_PREFIX to the ingress path, likely /streampark.

### CODE REVIEW

**Comment by chenshaojin** [comment]
cc @Al-assad

**Comment by Al-assad** [comment]
> cc @Al-assad
It seems that there are dead links in the README file.
image

**Review by Al-assad — DISMISSED** [review]
Thanks for your contribution, these codes look great 😃

File: deploy/helm/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+### 2. apply 
+
+```shell
+kubectl apply -f result/streampark/templates
+```
+
+### 3. open WebUI
+
+http://streampark.apache.org/streampark
```
Comment (wolfboys) [inline deploy/helm/README.md]: WebUI error, prompt the user to open the browser based on the actual URL.

File: deploy/helm/streampark/templates/streampark.yml [hunk 2]
```diff
@@ -57,11 +57,10 @@ spec:
               containerPort: {{ .Values.spec.containerPort }}
               protocol: TCP
           env:
-            - name: TZ
-              value: {{ .Values.timezone }}
+            {{- toYaml .Values.spec.container.env | nindent 12 }}
           securityContext:
             privileged: false
-          command: ["bash","-c","bash bin/startup.sh"]
+          command: ["bash","-c","bash ./bin/streampark.sh start_docker"]
```
Comment (wolfboys) [inline deploy/helm/streampark/templates/streampark.yml:63]: good job

**Comment by wolfboys** [comment]
cc @Al-assad PTAL , thx

**Review by Al-assad — APPROVED** [review]
LGTM

File: deploy/helm/README.md (hunk 1 above)
Comment (chenshaojin) [reply deploy/helm/README.md]: you are right

**Comment by chenshaojin** [comment]
Thank you for merging my code. ~v~

**Pull request merged**